app/mobile-content/exhibits/edit-exhibit-pages/edit-exhibit-pages.component.ts

```diff
@@ -19,78 +19,85 @@ import { Status } from '../../shared/status.model';
 })
 export class EditExhibitPagesComponent implements OnInit {
   @Input() exhibitId: number;
-  pages: ExhibitPage[];
+  infoPages = new Map<number, MobilePage>();
+  pages: MobilePage[];
   searchStatusOptions = Status.getValuesForSearch();
-  selectedStatus = 'ALL';
+  selectedStatus: statusTypeForSearch = 'ALL';
```

Comment:
I added the type here because otherwise TS would assume that `selectedStatus` is a simple string. Which it isn't.